instrumentation/spring/spring-webflux/spring-webflux-5.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/spring/webflux/v5_0/server/SpringWebfluxTest.java

```diff
@@ -138,11 +139,12 @@ void basicGetTest(Parameter parameter) {
                   span.hasKind(SpanKind.INTERNAL)
                       .hasParent(trace.getSpan(0))
                       .hasAttributesSatisfyingExactly(
+                          satisfies(CODE_FUNCTION, val -> val.isEqualTo("handle")),
                           satisfies(
-                              stringKey("spring-webflux.handler.type"),
+                              CODE_NAMESPACE,
                               parameter.annotatedMethod == null
                                   ? val -> val.contains(INNER_HANDLER_FUNCTION_CLASS_TAG_PREFIX)
-                                  : val -> val.isEqualTo(TestController.class.getName())));
```

Comment:
previously used `TestController` is more useful than `org.springframework.web.method.HandlerMethod` that is reported after these changes

Reply:
now it reports `TestController` as class name and method name as `handle`, `TestController` does not have this method